src/main/groovy/com/rundeck/plugins/ansible/ansible/AnsibleDescribable.java

```diff
@@ -527,4 +529,13 @@ public static String[] getValues() {
             .title("Encrypt Extra Vars.")
             .description("Encrypt the value of the extra vars keys.")
             .build();
+
+    Property YAML_DATA_SIZE_PROP = PropertyBuilder.builder()
+            .string(ANSIBLE_YAML_DATA_SIZE)
+            .required(false)
+            .title("Inventory Yaml Data Size")
+            .description("Set the MB size (Default value is 10MB)"+
```

Comment:
i think we could expand a little bit on the description, so customers wont get confused (since ansible allows inventories yaml, json and ini). Somehow specify that this is for the internal plugin process and not related to the ansible inventory type 🤔

Reply:
done... ✅